File: exs/mundo_1/javascript/022.js
/*
Desafio 022

Problema: Crie um programa que leia o nome completo de uma pessoa
          e mostre:
              - O nome com todas as letras maiúsculas e minúsculas.
              - Quantas letras ao todo(sem considerar espaços)
              - Quantas letras tem o primeiro nome.

Resolução do problema:
*/

const input = require('readline-sync').question //Chamando o modulo readline-sync para entrada de dados

//Entrada de dados
const nome = input('Digite seu nome: ')

//Declarando os dados no console "Saida de dados"
console.log(`Nome em maiúsculo ${nome.toUpperCase()}`)//Letras Maiúsculas
console.log(`Nome em minúsculo ${nome.toLowerCase()}`)//Letras Minúsculas
console.log(`Quantidade de letras  = ${nome.length}`)//Qtde de letras
console.log(nome.slice(0, 5).length) // Quantidade de letras do primeiro nome

File: exs/mundo_1/java/022.java
/*
    ### 022 - Analisador de textos
    Crie um programa que leia o nome completo de uma pessoa e mostre:
    em letras maiúsculas, minúsculas, a quantidade de letras sem espaço e quantas letras tem o primeiro nome
 */


import java.util.Locale;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        String nome;

        Scanner scanner = new Scanner(System.in);

        System.out.println("Digite seu nome: ");
        nome = scanner.nextLine();

        System.out.println("Maiusculas: " + nome.toUpperCase(Locale.ROOT));
        System.out.println("Minusculas: " + nome.toLowerCase(Locale.ROOT));
        System.out.println("Quantidade de letras sem espaço: " + nome.replaceAll(" ", "").length());
        System.out.println("Quantidade de lestras do primeiro nome: " + nome.split(" ")[0].length());

    }

}
